ruma-federation-api/src/openid/get_userinfo/v1.rs

```diff
@@ -0,0 +1,26 @@
+//! [GET /_matrix/federation/v1/openid/userinfo](https://matrix.org/docs/spec/server_server/r0.1.4#get-matrix-federation-v1-openid-userinfo)
+
+use ruma_api::ruma_api;
+use ruma_identifiers::UserId;
+
+ruma_api! {
+    metadata: {
+        description: "Exchanges an OpenID access token for information about the user who generated the token. Currently this only exposes the Matrix User ID of the owner.",
```

Comment:
We can omit this caveat (mostly so we don't have to remember to remove it if the endpoint changes :laughing:).

```suggestion
        description: "Exchanges an OpenID access token for information about the user who generated the token.",
```